### PR TITLE
furo: recipient, updating

### PR DESCRIPTION
**Pull request by LufyCZ**

### CODE REVIEW

File: apps/furo/components/AddressInput.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { isAddress } from '@ethersproject/address'
```
Comment (chillichelli) [inline apps/furo/components/AddressInput.tsx:1]: There is already an Ens input for this exact purpose. Checkout apps/furo/components/stream/CreateForm

```
<Web3Input.Ens
  value={value}
  onChange={onChange}
  error={!!error?.message}
  placeholder="Address or ENS Name"
/>
```

Comment (LufyCZ) [reply apps/furo/components/AddressInput.tsx:1]: Aw crap, didn't notice, will replace, thanks

Comment (LufyCZ) [reply apps/furo/components/AddressInput.tsx:1]: Can also just make the whole modal wider ig... @chillichelli

Comment (chillichelli) [reply apps/furo/components/AddressInput.tsx:1]: Hmmm, yeah first thought was truncate but that might not be the best solution in this particular case. Wider modal might not work well on mobile too so I guess smaller text is the only way

Comment (chillichelli) [reply apps/furo/components/AddressInput.tsx]: ahh yeah the wallet/bento buttons should have the offset as well

Comment (chillichelli) [reply apps/furo/components/AddressInput.tsx]: This looks good tho, love it!

Comment (chillichelli) [reply apps/furo/components/AddressInput.tsx]: The loader could still be there tho but absolute positioned, and just hide when loading is done

**Pull request merged**